### PR TITLE
Fixes issue #28 - Nested accordion

### PR DESCRIPTION
**Pull request by kdo**
### Description of the Change
Change selector so that only direct descendants of the accordion area are selected, allowing nested accordions.

### Benefits
Allowing nested accordions

### Verification Process
- Tested in IE11, Edge, Safari, Chrome, Firefox
- Tried to click to toggle accordions: pass
- Used keyboard to navigate: pass
- Used keyboard to toggle accordions: pass

The nested accordion is correctly contained.

### Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.

### Applicable Issues

Issue #28

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -16,9 +16,25 @@ <h2 class="accordion-label">Accordion Heading</h2>
 				<p>here the content of 1st tab <a href="#">link</a></p>
 			</div> <!-- //.accordion-content -->
 
-			<button class="accordion-header" type="button">Accordion Header</button>
+			<button class="accordion-header" type="button">Accordion Header with Nested Accordion</button>
 			<div class="accordion-content">
 				<p>here the content of 2nd tab <a href="#">link</a></p>
```
Comment (joesnellpdx) [inline demo/index.html:22]: @kdo 
Only thing I'd change is adding  something like `<h2 class="accordion-label" tabindex="-1">Parent Accordion Heading</h2>` here for consistency.

**Review by joesnellpdx — CHANGES_REQUESTED** [review]
@kdo 

One minor change. Looks great!
I have to question the use case of a nested accordion, but conversation is for another day.

File: demo/index.html (hunk 1 above)
Comment (kdo) [reply demo/index.html:22]: Thanks @joesnellpdx added!

**Review by joesnellpdx — APPROVED** [review]
Looks great! Approved

**Pull request merged**